rewrite-java/src/main/java/org/openrewrite/java/JavaParser.java

```diff
@@ -57,6 +57,18 @@
                 .collect(toList());
     }
 
+    /**

```

Comment:
Currently,` buildSnippet` was constructing a parser that was only aware of core Java library types. Introducing this method to help construct a parser that is aware of the entire runtime classpath.